### DISCUSSION TITLE
Problemi dopo l'aggiornamento all'ultima versione

### DISCUSSION THREAD

**Discussion by MValente028** [globaleaks/globaleaks-whistleblowing-software#3904] [General]
Buonasera,
da quando ho aggiornato sulla mia macchina UBUNTU all'ultima versione di GlobaLeaks non riesco piu' a raggiungere la web interface in localhost, tramite IP (della macchina ubuntu ovviamente) nel browser rete viene effettuato un redirect automatico al dominio dell'hostname che avevo inserito ma continua ovviamente a non visualizzare l'interfaccia, in precedenza già mi è capitato, ma bastava che entravo sulla macchina UBUNTU in localhost sul browser e riuscivo tranquillamente ad accedere all'interfaccia, ora quando provo, mi restituisce Connessione non riuscita.

**Comment by evilaliv3** [comment]
GlobaLeaks nelle recenti versioni risponde su porte 80 e 443 che sono puntate tramite iptables sulle port 8080 e 8443.

Vi suggeriamo di verificare se riuscite a raggiungere il sistema su 0.0.0.0 / porta 443 o su 127.0.0.1 porta 8443

**Comment by MValente028** [comment]
No, la connessione mi viene rifiutata su entrambe le porte.

Il giorno mar 19 dic 2023 alle ore 18:13 Giovanni Pellerano <
***@***.***> ha scritto:
